api/client/cluster/client.go

```diff
@@ -192,3 +194,78 @@ func (c *clusterClient) EraseAlert(resource api.ResourceType, alertID int64) err
 	}
 	return nil
 }
+
+func (c *clusterClient) SetDefaultSecretKey(secretKey string, override bool) error {
+	reqBody := &secrets.DefaultSecretKeyRequest{
+		DefaultSecretKey: secretKey,
+		Override:         override,
+	}
+	path := clusterPath + secretPath + "/defaultsecretkey"
+	request := c.c.Put().Resource(path).Body(reqBody)
+	resp := request.Do()
+	if resp.Error() != nil {
+		return resp.FormatError()
+	}
+	return nil
+}
+
+func (c *clusterClient) GetDefaultSecretKey() (interface{}, error) {
+	var defaultKeyResp interface{}
+	path := clusterPath + secretPath + "/defaultsecretkey"
+	request := c.c.Get().Resource(path)
+	err := request.Do().Unmarshal(defaultKeyResp)
+	if err != nil {
+		return defaultKeyResp, err
+	}
+	return defaultKeyResp, nil
+}
+
+func (c *clusterClient) Set(secretID string, secretValue interface{}) error {
```

Comment:
Cannot use names `Set` and `Get` for `clusterClient` because it is too generic, when the API is only affecting the secret. I think this is supposed to be `SecretSetValue` and `SecretGetValue`

Reply:
Actually, preface all of these API calls with the `SecretXXX` name... That will group them into that "namespace". So it would look like `CloudBackupXXX` functions in `client/volume/client.go`

Reply:
@piyush-nimbalkar , @adityadani should I change interface Set and Get back to SetValue/GetValue?